data/hai/hai reveal 4 middle.txt

```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	A moment later Sayari squeaks and looks up in horror. "You triggered an election?! This wasn't just making it public, you had the petition for redress ready to go all along!"`
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hoped would happen, isn't it?"`
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
-			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. "Captain <last>, you must leave immediately. Get on your ship and fly to <planet> right now! The winds of change are afoot and we will have an election. They'll hit the threshold in maybe only twenty minutes, and it'll take maybe thirty more for the bureaucracy to act on it. The instant the election is announced formally, all non-essential ships will be grounded, all communication in or out of Hai space will be cut off, and the border will be sealed to everyone - not in the polite way either. If even one ship tries to cross the border to investigate... there's no telling what will happen."`
+			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. They are ready to send a fleet to human space and extinguish the pirate threat. Will you do what is right for our people and negotiate this with the humans, or will you be replaced?"`
+			`	At the word "replaced," murmurs rise, and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify.`
+			`	"Captain <last>, can you speak to the truth of these disappearances?"`
+			choice
+				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
+					goto uncertainties
+				`	"It is a coordinated kidnapping campaign, and we must put a stop to it but take care not to start a war."`
```

Comment:
These are both comparatively a lot of words for <first> to say in a choice.
They should be re-assessed when the preceding text changes.